packages/material-ui-lab/src/Skeleton/Skeleton.js

```diff
@@ -31,6 +31,7 @@ export const styles = theme => ({
   /* Styles applied to the root element if `disabledAnimate={false}`. */
   animate: {
     animation: '$animate 1.5s ease-in-out infinite',
+    animationDelay: '0.5s',
```

Comment:
What about using the short hand notation with the animation prop?

Reply:
I have added the 0.5s to the animation shorthand. Is it possible to pass a prop to the styles const?